lib/components/Button/Button.tsx

```diff
@@ -24,12 +25,14 @@ export const Button = ({
   children,
   className,
   loading,
+  loadingText = 'Loading ...',
   ...rest
 }: ButtonProps) => {
   if (loading) {
     return (
       <ButtonBase
         variant={variant}
+        aria-busy
```

Comment:
Shouldn't `aria-busy` be conditional on whether the button is actually loading?
```suggestion
        aria-busy={loading}
```

Reply:
Wait, nvm I missed that this entire block is conditional on the loading prop 😅
Sorry!